tool-nextjs-starter/src/auth.ts

```diff
@@ -0,0 +1,21 @@
+import { authHandler, AuthHandlerParams, sessionCookieStore } from '@storyblok/app-extension-auth';
+
+['APP_CLIENT_ID', 'APP_CLIENT_SECRET', 'APP_URL'].forEach((key) => {
+    if (!process.env[key]) {
+        throw new Error(`Environment variable "${key}" is missing.`);
```

Comment:
Maybe we could add more message about `.env.local.example`?

Reply:
I think as long as we put it clearly into the readme it should be enough.